cinema/views.py

```diff
@@ -0,0 +1,43 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.shortcuts import get_object_or_404
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from rest_framework import status
+
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+def create_movie_or_raise(data, instance=None):
+    if instance:
+        serializer = MovieSerializer(instance, data)
+    else:
+        serializer = MovieSerializer(data=data)
+    serializer.is_valid(raise_exception=True)
+    serializer.save()
+    return Response(serializer.data, status=status.HTTP_201_CREATED)
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
```

Comment:
There is no need for `else` since the code will not execute after the `return` statement anyway. Please, remove the redundant `else`